components/skeleton/skeleton-mixin.js

```diff
@@ -27,7 +27,7 @@ export const skeletonStyles = css`
 	}
 	:host([skeleton]) .d2l-skeletize {
 		background-color: transparent;
-		border-color: transparent;
+		border-color: var(--d2l-color-sylvite);
```

Comment:
@mdgbayly One more adjustment to this, so now it's gone from `none` -> `transparent` -> `sylvite`. Without this, components that do have borders (like checkboxes) end up looking like their skeletons have shrunk by the border width. By only setting the colour, elements that don't have borders won't be impacted.